### DISCUSSION TITLE
Boolean Operation Crashes Program

### DISCUSSION THREAD

**Discussion by qferreira-1** [lassoan/SlicerBoneReconstructionPlanner#106] [General]
Hey All,

So I’ve noticed this the last few cases where I would get to the Boolean operation of the fibula guide or mandible guide to create the guide and th e program will crash. I’ve tried this with simple reconstructions as well and have noticed I’ve had to package the file, open in 5.03 or 5.2.2 to complete the operation. 
there are no error messages that I can see it just simply closes the program. 

Let me know if this is something you have experienced or if there is any ideas you may have that is causing this. Thanks so much for your help.
Best,
Q

**Comment by mauigna06** [comment]
@qferreira-1 thanks for reporting. I'll check when I have time. I guess you can use the earlier Slicer version as a workaround for the moment. 
Please open a new issue next time instead of a discussion since they are commonly used to report bugs :)

**Reply by qferreira-1** [reply]
Apologies, will do so next time. - Side request, is there a way to get the dental implant branch on Slicer 5.2.2?

**Reply by mauigna06** [reply]
I think you should be available to click "Update extension" for BoneReconstructionPlanner in the Extensions Manager. Let me know if that works :)

**Reply by qferreira-1** [reply]
Awesome, it works. Thanks for your help! We will be trialling this for a case on Monday next week.

**Reply by mauigna06** [reply]
Great. Please open a discussion to let us know how it goes :)

**Comment by drmaxface** [comment]
Hello, I've been trying to learn how to use your Slicer module but these crashes still seem to be a problem after making boolean operations. I haven't tried older slicer version yet (currently using 5.4.0) is that workaround still usable or any other way to solve this issue ? Thanks in advance

**Reply by mauigna06** [reply]
Hi @drmaxface 
You need Slicer 5.2.2 Stable currently.
Please see:
https://github.com/SlicerIGT/SlicerBoneReconstructionPlanner#installing-bonereconstructionplanner

**Reply by drmaxface** [reply]
thank you again
